kivy/uix/camera.py

```diff
@@ -90,7 +90,9 @@ def __init__(self, **kwargs):
         fbind('resolution', on_index)
         on_index()
 
-    def on_tex(self, *l):
+    def on_tex(self, camera):
+        self.texture = texture = camera.texture
+        self.texture_size = list(texture.size)
```

Comment:
You don't have to convert to `list` because `texture_size` is a `ListProperty` and assigning to it will convert `texture.size` to `ObservableList`.

Reply:
Actually, you don't have to set `texture_size` at all as it will be updated by `on_texture` method from `Image` class.

Reply:
You can safely remove line `self.texture_size = list(texture.size)` from `on_tex` and `_camera_loaded` methods.